openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/upsell.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
```

Comment:
nit: maybe name this file "upsell_cta.html" to match the other one?

Reply:
sure.  renamed.